lane.py

```diff
@@ -726,7 +726,10 @@ def from_request(cls, library, config, get_argument, get_header, worklist,
         if media not in EditionConstants.KNOWN_MEDIA:
             media = None
         extra['media'] = media
-        extra['languages'] = languages
+        languageQuery = get_argument("language", None)
+        if languageQuery != "all" :
+            extra['languages'] = languages
```

Comment:
I'm not sure how you're using this, but I would think if the `languageQuery` had some languages that weren't "all", you should use them here instead of the languages from the header. If `languageQuery` is `None`, you could use the header, and if it's "all", you could not use anything.

Reply:
Maybe add a comment that the only possible value is "all" then. This is confusing if you look at this code by itself. Eventually I think we will want to have something like what I described.